Docs/source/running_cpp/profiling.rst

```diff
@@ -1,46 +1,35 @@
+Load balancing can be activated in WarpX by setting the input parameter `warpx.load_balance_int <https://warpx.readthedocs.io/en/latest/running_cpp/parameters.html#distribution-across-mpi-ranks-and-parallelization>`__.
 
-  ::
-
-     module swap craype-haswell craype-mic-knl
-     make -j 16 COMP=intel USE_VTUNE=TRUE
-
-  (where the first line is only needed for KNL)
-
-- In your SLURM submission script, use the following
-  lines in order to run the executable. (In addition
-  to setting the usual ``OMP`` environment variables.)
+Profiling with AMReX's built-in profiling tools
+-----------------------------------------------
 
-  ::
+By default, WarpX uses the AMReX baseline tool, the TINYPROFILER, to evaluate the time information for different parts of the code (functions) between the different MPI ranks.
+The results, timers, are stored into four tables in the standard output, stdout, that are located below the simulation steps information and above the warnings regarding unused input file parameters (if there were any).
 
-     module load advisor
-     export ADVIXE_EXPERIMENTAL=roofline
-     srun -n <n_mpi> -c <n_logical_cores_per_mpi> --cpu_bind=cores advixe-cl -collect survey -project-dir advisor -trace-mpi -- <warpx_executable> inputs
-     srun -n <n_mpi> -c <n_logical_cores_per_mpi> --cpu_bind=cores advixe-cl -collect tripcounts -flop -project-dir advisor -trace-mpi -- <warpx_executable> inputs
+The timers are displayed in tables for which the columns correspond to:
 
-  where ``<n_mpi>`` and ``<n_logical_cores_per_mpi>`` should be replaced by
-  the proper values, and ``<warpx_executable>`` should be replaced by the
-  name of the WarpX executable.
+* name of the function
+* number of times it is called in total
+* minimum of time spent exclusively/inclusively in it, between all ranks
+* average of time, between all ranks
+* maximum time, between all ranks
+* maximum percentage of time spent, accross all ranks
 
-- Launch the Intel Advisor GUI
+If the simulation is well load balanced the minimum, average and maximum times should be identical.
 
-  ::
+The top two tables refer to the complete simulation information.
+The bottom two are related to the Evolve() section of the code (where each time step is computed).
 
-     module load advisor
-     advixe-gui
+Each set of two timers show the exclusive, top, and inclusive, bottom, information depending on wether the time spent in nested sections of the codes are included.
 
-  (Note: this requires to use ``ssh -XY`` when connecting to Cori.)
+For more detailed information please visit the `AMReX profiling documentation <https://amrex-codes.github.io/amrex/docs_html/AMReX_Profiling_Tools_Chapter.html>`__.
 
+.. node:
+   We encourage all users to include the TINYPROFILERS simulation information in their github issues, if they verified strong load imbalance.
```

Comment:
I am not sure I agree with that, the user should only include performance data if the issue is about code performance.

Reply:
Oh you are totally right. In my mind I was assuming that if a user even pays attention to the load balancing, it is because they are concerned about the performance. But I was thinking wrongly.

Reply:
I can't decide between removing the note or changing it to something like:
```suggestion
   We encourage all users who need to improve simulations performance (to run very large/complex configurations that might face strong load imbalance), to include their runs' TINYPROFILERS information in their github issues.
```
Of course, if/when we do issue templates we can include this suggestion there in a more summarized way.